src/write/elf/object.rs

```diff
@@ -771,7 +771,7 @@ impl<'a> Object<'a> {
                     SectionKind::Tls => elf::SHF_ALLOC | elf::SHF_WRITE | elf::SHF_TLS,
                     SectionKind::UninitializedData => elf::SHF_ALLOC | elf::SHF_WRITE,
                     SectionKind::UninitializedTls => elf::SHF_ALLOC | elf::SHF_WRITE | elf::SHF_TLS,
-                    SectionKind::ReadOnlyData => elf::SHF_ALLOC,
+                    SectionKind::ReadOnlyData | SectionKind::ReadOnlyDataWithRel => elf::SHF_ALLOC,
```

Comment:
`ReadOnlyDataWithRel` should get the `SHF_WRITE` flag, right?

Reply:
Yeah, I think so, let me update that.